subxt/src/tx/tx_client.rs

```diff
@@ -417,12 +420,13 @@ where
     pub fn from_bytes(client: C, tx_bytes: Vec<u8>) -> Self {
         Self {
             client,
+            encoded_prefixed: tx_bytes.encode(),
```

Comment:
The bytes provided from eg line 394 are already prefixed with length, so we shouldn't need to do this again I think?